test/cmdlineTests/viair_subobject_optimization/output

```diff
@@ -0,0 +1,8 @@
+
+======= viair_subobject_optimization/input.sol:C =======
+Binary:
+60808060405234605357601f60cf38819003918201601f19168301916001600160401b03831184841017605957808492602094604052833981010312605357604690516073565b604051604d908160828239f35b50600080fd5b50505050634e487b7160e01b600052604160045260246000fd5b60be600160481b031460535756fe6080604052600080fdfea26c6578706572696d656e74616cf564736f6c63782b302e382e31332d646576656c6f702e323032322e332e372b636f6d6d69742e37633931646430352e6d6f640041
+
+======= viair_subobject_optimization/input.sol:D =======
+Binary:
+6080806040523461001657610203908161001d8239f35b50600080fdfe6080806040526004361015610015575b50600080fd5b600090813560e01c6326121ff01461002d575061000f565b3461009657816003193601126100965760ce603f8101601f1916820167ffffffffffffffff81118382101761007f5761007b93506040528082526100f260208301396040519182918261009a565b0390f35b5050634e487b7160e01b8252506041600452602490fd5b5080fd5b919091602080825283519081818401526000945b8286106100db5750508060409394116100ce575b601f01601f1916010190565b60008382840101526100c2565b85810182015184870160400152948101946100ae56fe6080806040523460525760ce80380390601f19601f8301168301918383106001600160401b03841117605857808492602094604052833981010312605257604590516072565b604051604d908160818239f35b50600080fd5b50505050634e487b7160e01b600052604160045260246000fd5b60be600160481b031460525756fe6080604052600080fdfea26c6578706572696d656e74616cf564736f6c63782b302e382e31332d646576656c6f702e323032322e332e372b636f6d6d69742e37633931646430352e6d6f640041a26c6578706572696d656e74616cf564736f6c63782b302e382e31332d646576656c6f702e323032322e332e372b636f6d6d69742e37633931646430352e6d6f640041
```

Comment:
Note that on develop this contains ``ffffffffffffffff42``, while the one for ``C`` doesn't.